tests/TimePicker.spec.jsx

```diff
@@ -243,5 +243,26 @@ describe('TimePicker', () => {
         done();
       });
     });
+
+    it('close panel correctly on ENTER', (done) => {
+      const picker = renderPicker();
+      const input = TestUtils.scryRenderedDOMComponentsWithClass(picker,
+        'rc-time-picker-input')[0];
```

Comment:
Do you mean `rc-time-picker-panel-input`?

Reply:
Actually no. But maybe I should?

Reply:
When panel is open, `rc-time-picker-input` will be covered by the panel, then how can you press <kbd>Enter</kbd> on `rc-time-picker-input`?

Reply:
I need to check this a bit later. I thought open/close events are processed in a parent module. Thanks for the note!

Reply:
You're right. The first implementation works only on open panel with no focus there through Picker.onKeyDown.
The second test I've just added checks Enter key processing on focused header. The Header is also updated with onEnter prop from Picker.